### DISCUSSION TITLE
yt-dlg 1.4.0

### DISCUSSION THREAD

**Discussion by oleksis** [oleksis/youtube-dl-gui#16] [Announcements]
* Add initial support to Dark Theme.
* Update locales en_US, es_ES .
* Replace mainframe.CustomComboBox for mainframe.ListBoxComboPopup.
* Fix default video format selection in mainframe.py
* Remove twodict Dependency

<hr /><em>This discussion was created from the release <a href='https://github.com/oleksis/youtube-dl-gui/releases/tag/v1.4.0'>yt-dlg 1.4.0</a>.</em>

**Comment by oleksis** [comment]
## Supporting Dark Theme

![yt-dlg_main](https://user-images.githubusercontent.com/44526468/117546401-3a4ff180-aff8-11eb-9fb9-163ae649c523.png)
![yt-dlg_options_dark](https://user-images.githubusercontent.com/44526468/117546403-3c19b500-aff8-11eb-8426-aa1d7df78908.png)

**Comment by nicolaasjan** [comment]
Maybe make the white parts greyish and add more contrast in the buttons below?

**Reply by oleksis** [reply]
Would you recommend an RGB color for the gray toning and button contrast?

**Reply by nicolaasjan** [reply]
I'm not very good at designing things, haha.

Regarding the buttons, maybe make them something like in the program [Axiom](https://axiomui.github.io/).

![axiom](https://user-images.githubusercontent.com/14093220/117566763-dfca9a00-b0b8-11eb-8c69-863f84dc13e6.png)

**Reply by github-account1111** [reply]
Something neutral and standardized like [Tomorrow Night](https://github.com/chriskempson/tomorrow-theme#tomorrow-night) or [Monokai](https://monokai.pro).

**Reply by oleksis** [reply]
@nicolaasjan and @github-account1111 Thanks for share! I so glad, I will check your advice 🙏🏾
